list/LIST_CONSULTATION_README.md

```diff
@@ -0,0 +1,211 @@
+| orderedBy.coding\[0].system      | fixed value = `http://hl7.org/fhir/list-order`                                                                                                                                                  |
+| orderedBy.coding\[0].code        | fixed value = `system`                                                                                                                                                                          |
+| orderedBy.coding\[0].display     | fixed value = `Sorted by System`                                                                                                                                                                |
+| encounter                        | reference to mapped [Encounter](../practioners/README.md)                                                                                                                                       |
+| entry[index].item.reference      | reference to one or more mapped [List (Topic)](./LIST_TOPIC_README.md)                                                                                                                          |
+
+The following List fields are not currently populated by the adaptor:
+- identifier
+- source
+- note
+- entry.flag
+- entry.deleted
+- entry.date
+
+
+<details>
+    <summary>Example JSON</summary>
+
+```
+{
+    "resource": {
+        "resourceType": "List",
+        "id": "1E8A8448-A0C1-11ED-808B-AC162D1F16F0-CONS",
+        "meta": {
+            "profile": [
+                "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-List-1"
+            ]
+        },
+        "status": "current",
+        "mode": "snapshot",
+        "title": "Surgery Consultation Note",
+        "code": {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "325851000000107",
+                    "display": "Consultation"
+                }
+            ]
+        },
+        "subject": {
+            "reference": "Patient/14013417-5eb8-4fb2-9916-4c1621e2533b"
+        },
+        "encounter": {
+            "reference": "Encounter/1E8A8448-A0C1-11ED-808B-AC162D1F16F0"
+        },
+        "date": "2010-12-16",
+        "orderedBy": {
+            "coding": [
+                {
+                    "system": "http://hl7.org/fhir/list-order",
+                    "code": "system",
+                    "display": "Sorted by System"
+                }
+            ]
+        },
+        "entry": [
+            {
+                "item": {
+                    "reference": "List/5a8d2ec6-807d-4db5-b6c0-a757bbfb5372"
+                }
+            }
+        ]
+    }
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+A `List (Consultation)` is mapped from references `resource` with a `resource.type` of `list` when mapping an [Encounter](../encounters/README.md) to an `EhrComposition` from a GP Connect FHIR `Encounter`
```

Comment:
Discussed being clearer that the `List (Consultation)` doesn't generate any XML itself, but is used to identify which Topics to then insert into the `Encounter`.

Reply:
Completed